components/gitpod-db/src/typeorm/entity/db-team-subscription.ts

```diff
@@ -51,4 +51,9 @@ export class DBTeamSubscription implements TeamSubscription {
         default: false
     })
     deleted?: boolean;
+
+    @Column({
+        default: false
+    })
+    noMoreResources?: boolean;
```

Comment:
I feel like I always point out the silly things, but, as always, I'm not sure about the naming. 😅

- `noMoreResources` sounds a bit confusing (e.g. it means the team doesn't have any resources anymore? 🤔)

How about something like:

- `noPremiumResources`
- `noExtraResources`
- `exemptFromPremiumResources`
- ... ?

Reply:
Yeah, I feel you. I'm no fan of it, too. But it fits with other usage in our code-base, especially the relevant marker on clusters.

I also wondered if it make sense to make this field a generic JSON bag, with this field just being one of multiple possible tags. WDYT? :thinking:  This way we could more easily adjust once we explored what we want to support beyond "more-resources".

Reply:
> Yeah, I feel you. I'm no fan of it, too. But it fits with other usage in our code-base, especially the relevant marker on clusters.

Ah 😬 I wasn't aware of these. Still, I don't think team DB should strictly follow the naming conventions of cluster DB (they're somewhat unrelated), and in any case, I think it's always better to use clear self-evident naming as opposed to following "unfortunate" per-existing naming conventions. 🙈

Reply:
> I also wondered if it make sense to make this field a generic JSON bag, with this field just being one of multiple possible tags. WDYT? 🤔

I think this would be premature optimization:
- We don't know yet whether we'd even use that
- The `d_b_team_subscription` is about to get deprecated anyway
- Aren't JSON bags done to avoid significant table modifications for tables with many rows? (E.g. would be okay with `d_b_workspace_instance`, but I don't think altering `d_b_team_subscription` is significant performance-wise)

Reply:
I renamed to `excludeFromMoreResources` to better fit into the `TeamSubscription` perspective/model.

> I think this would be premature optimization:

True. Alos, really hope we can get rid of it rather sooner than later. To retiterate: The whole point of this is to avoid the (tremendous) overhead of (re-)adding a distinct Chargebee subscription for these kind of clients. We had that once, but decided to drop it... :confused: